tools/topology/topology2/CMakeLists.txt

```diff
@@ -3,7 +3,15 @@
 # 3.17 required: foreach(<loop_var>... IN ZIP_LISTS <lists>)
 cmake_minimum_required(VERSION 3.17)
 
-add_custom_target(topologies2)
+add_custom_target(topologies2 ALL
+    # Create directory structure to be provided for deployment
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/target/sof-ipc4-tplg
+    COMMAND ${CMAKE_COMMAND} -E create_symlink sof-ipc4-tplg ${CMAKE_CURRENT_BINARY_DIR}/target/sof-ace-tplg
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/target/development
+    # copy the topology files only to target
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different production/*.tplg ${CMAKE_CURRENT_BINARY_DIR}/target/sof-ipc4-tplg/
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different development/*.tplg ${CMAKE_CURRENT_BINARY_DIR}/target/development/
+)
```

Comment:
Seems to do the trick. Given the baseline has no install logic at all, this seems like a huge value-add. 

It would seem the dependencies are also set for topologies2 target, so good in that sense as well.

Reply:
So this is basically a DIY "install". I suspect there is a slightly more "native" and performant way to do this but this is very simple and easy to maintain later so plenty good enough to get started.